website/src/components/Chat/ChatMessageEntry.tsx

```diff
@@ -175,6 +177,12 @@ export const ChatMessageEntry = memo(function ChatMessageEntry({
                 )}
                 {state === "complete" && (
                   <>
+                    <BaseMessageEmojiButton
```

Comment:
Can you adjust icon base on the `isPlainText` state? I think using these two icons will make more sense
 https://tabler-icons.io/i/markdown-off
 https://tabler-icons.io/i/markdown

Reply:
noted. have made the relevant changes.